nessie/src/main/java/org/apache/iceberg/nessie/NessieIcebergClient.java

```diff
@@ -523,4 +528,49 @@ public void close() {
       api.close();
     }
   }
+
+  public TableMetadata loadTableMetadataWithNessieSpecificProperties(
```

Comment:
This method was moved from `NessieTableOperations`
- Reference is no longer passed as it is available locally. 
- Some other required params are passed newly.